src/data_types/schema.rs

```diff
@@ -17,6 +17,7 @@ pub struct Schema {
     pub issuer_id: IssuerId,
 }
 
+// QUESTION: If these must be unique, why not directly store them as a set?
```

Comment:
Sets are unordered and some ledgers create their identifier from a hash of the content, i.e. ordering matters.

Reply:
I see, fair enough. Might be worth looking into a `BTreeSet` in the future (which is ordered).